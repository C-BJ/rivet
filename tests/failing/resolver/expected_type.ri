fn my_fn() { }

fn main() {
    let x: my_fn = ();
}
