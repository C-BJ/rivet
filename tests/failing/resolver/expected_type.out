tests/failing/resolver/expected_type.ri:4:12: error: expected type, found function
    |
  4 |     let x: my_fn = ();
    |            ^
rivetc: error: could not compile package `expected_type`, aborting due to previous error.